modules/cnf-image-based-upgrade-backup-guide.adoc

```diff
@@ -20,7 +20,7 @@ The following resources must be excluded from the backup:
 
 There are two local storage implementations for {sno}:
 
-Local Storage Operator (LSO):: The {lcao} backs up and restores the required artifacts, including `LocalVolumes` and their associated `StorageClasses`. You must exclude the `persistentVolumes` resource in the application `Backup` CR.
+Local Storage Operator (LSO):: The {lcao} automatically backs up and restores the required artifacts, including `localvolume` resources and their associated `StorageClass` resources. You must exclude the `persistentVolumes` resource in the application `Backup` CR.
```

Comment:
I think we want to change `persistentVolumes` to `persistentvolumes` in the backup CR example 
```suggestion
Local Storage Operator (LSO):: The {lcao} automatically backs up and restores the required artifacts, including `localvolume` resources and their associated `StorageClass` resources. You must exclude the `persistentvolumes` resource in the application `Backup` CR.
```